ovirt-ansible-vm-infra.spec.in

```diff
@@ -14,7 +14,7 @@ Group:          Virtualization/Management
 BuildArch:      noarch
 Url:            http://www.ovirt.org
 
-Requires: ansible >= 2.9.0
+Requires: ansible >= 2.9.0, python3-jmespath
```

Comment:
Please put python3-jmespath on separate line and IMO it's worth also to split it to support also EL7:

```
%if 0%{?rhel} <= 7
Requires	python-jmespath
%else
Requires	python3-jmespath
%endif
```